cmd/bnsd/client/testdata/genesis_json.md

```diff
@@ -20,13 +20,10 @@ A is an address for a public key or contract with currency tokens filled.
       "address": "E28AE9A6EB94FC88B73EB7CBD6B87BF93EB9BEF0",
       "coins": [
         {
-          "ticker": "IOV",
-          "whole": 123456789
+          "ticker": "123456789 IOV"
```

Comment:
This was not tested, was it?

Reply:
Unfortunately I can not test this with the genesis.json file in this directory as the client uses a different model for deserialization. I had reverted the changes there but these 2 lines are just wrong. Good finding!